source/calendar/calendar-styles.css

```diff
@@ -21,27 +21,51 @@ body {
 
 .main-container {
     display: flex;
+    flex-wrap: wrap;
     justify-content: center;
     align-items: center;
     margin-right: 50px;
+    flex-direction: row;
+    background-color: lightgray;
+    gap: 10px;
 }
 
 .calendar-container {
     flex: 3;
+    min-width: 300px;
+    max-width: 600px;
+}
+
+.calendar-container, .todo-list-container {
+    flex: 1 0 300px;
+    max-width: 600px;
 }
 
-@media (width <= 768px) {
+/* MOBILE RESPONSIVENESS BREAKPOINT - SMALLER SCREEN SIZE */
```

Comment:
Can you add in the comment if this is the vertical stack? Describe why we have the breakpoint (ie make the buttons into a column instead of a row)